requirements.txt

```diff
@@ -1,3 +1,28 @@
-bokeh>=2.3.1
-descartes>=1.1.0
-Shapely>=1.7.1
+#
+# This file is autogenerated by pip-compile with python 3.7
+# To update, run:
+#
+#    pip-compile
+#
+bokeh==2.4.2
+    # via -r requirements.in
+jinja2==3.0.3
+    # via bokeh
+markupsafe==2.0.1
+    # via jinja2
+numpy==1.21.5
+    # via bokeh
+packaging==21.3
+    # via bokeh
+pillow==8.4.0
+    # via bokeh
+pyparsing==3.0.6
+    # via packaging
+pyyaml==6.0
+    # via bokeh
+shapely==1.8.0
```

Comment:
This version doesn't exist on Mac, but you're not supporting Mac at the moment, and there are also issues. I may take a stab at Mac support in a separate PR if you don't mind.

Reply:
Let me add this concern to the README.